### PR TITLE
Host macs from fru

### PR DESCRIPTION
**Pull request by nest1ing**
This brings functionality for fetching x722 MAC addresses from FRU via D-Bus and storing them to the host flash.

### CODE REVIEW

File: src/nvm_x722.cpp [hunk 1]
```diff
@@ -167,3 +171,161 @@ NvmX722::word_t NvmX722::calcChecksum() const
...
+        }
+
+        return bi;
+    }
+} __attribute__((packed));
+
+using Property = std::variant<std::string>;
+using Properties = std::map<PropertyName, Property>;
+/**
+ * @brief Get all D-Bus properties from specifice service.
```
Comment (AlexanderAmelkin) [inline src/nvm_x722.cpp]: 'a specific' ? 'the specified' ?

File: src/nvm_x722.cpp [hunk 2]
```diff
@@ -167,3 +171,161 @@ NvmX722::word_t NvmX722::calcChecksum() const
...
+ *
+ * NOTE: this not in dbus.hpp as we expects the string only properties.
```
Comment (AlexanderAmelkin) [inline src/nvm_x722.cpp]: `as we expect only the string properties` ? Anyway, I can't see how this justifies this not being in `dbus.hpp`.

**Review by AlexanderAmelkin — CHANGES_REQUESTED** [review]

File: src/nvm_x722.cpp (hunk 1 above)
Comment (nest1ing) [reply src/nvm_x722.cpp]: fixed

File: src/nvm_x722.cpp (hunk 2 above)
Comment (nest1ing) [reply src/nvm_x722.cpp]: fixed.

File: src/nvm_x722.cpp (hunk 1 above)
Comment (AlexanderAmelkin) [reply src/nvm_x722.cpp]: Nope. There is not such word `specificied` :).

Comment (nest1ing) [reply src/nvm_x722.cpp]: refixed :)

**Review by AlexanderAmelkin — APPROVED** [review]

**Pull request merged**